### PR TITLE
Added New Event: IstioCon Virtual 2023

### LINKED ISSUE #4756: [Event] Add new event IstioCon Virtual 2023
Please add the following event to layer5.io/events.

**Event:** IstioCon Virtual 2023
- Event Title: Multiplayer Istio: Collaborative WASM Plugins with Intel and Layer5
- Event Link: https://events.linuxfoundation.org/istiocon/
- Talk Description: Join Layer5 and Intel at the Istio conference on Monday, Sept. 25th to Tuesday, Sep. 26th. IstioCon is a 100% virtual event that is designed to connect community members across the globe with Istio and the Istio ecosystem.
- Registration Link: https://cvent.me/LnMXkN
- Speakers: Lee Calcote, Xin Huang
- Thumbnail image - 
Untitled

- Deck: 
- Video: 

---
<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by malaviya-parth**
**Description**

This PR fixes #4756

**Notes for Reviewers**
- Have added a link tag register here below the text
- Image resolution is the default provided

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Chadha93** [comment]
Thanks for the PR @malaviya-parth. 

Will be merged after the 21st.

**Comment by l5io** [comment]
🚀 Preview for commit b6f140e09142b1f24918c6fd50b56b0566f4a0ba at: https://64dfab776d07a002ea7bce85--layer5.netlify.app

**Review by hirentimbadiya — CHANGES_REQUESTED** [review]
@malaviya-parth, please undo the changes you made in package-lock.json file.

**Comment by l5io** [comment]
🚀 Preview for commit 336d5b14c81007ff8059cac9fca0ff36882fba3f at: https://64e059acc07fa023080ce179--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
@malaviya-parth @Chadha93 

The PR looks good however a few enhancement needs to be done on the thumbnail image
image

//
If we can alter-image , otherwise everything is Good⭐
It would be nice if the thumbnail image is 
image

**Comment by malaviya-parth** [comment]
@Ghat0tkach 
So should I change it with the image you provided? or wait for @Chadha93 's confirmation?

**Comment by Ghat0tkach** [comment]
@malviya-parth lets wait for his confirmation😅

**Comment by Ghat0tkach** [comment]
hey @malaviya-parth 

Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Comment by malaviya-parth** [comment]
> hey @malaviya-parth
> 
> Let's discuss it on the websites call. Please add this as an agenda item in the meeting minutes, if you would :)
> 
> https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

Sorry I was not able to make it to the meeting due to some work, If any changes are required can you tell me. Once Again sorry for not attending.

**Comment by Chadha93** [comment]
> > hey @malaviya-parth
> > Let's discuss it on the websites call. Please add this as an agenda item in the meeting minutes, if you would :)
> > [docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima)
> 
> Sorry I was not able to make it to the meeting due to some work, If any changes are required can you tell me. Once Again sorry for not attending.

@malaviya-parth, Will you please update the thumbnail with the image @Ghat0tkach, provided?

**Comment by l5io** [comment]
🚀 Preview for commit 637e02fdba75b4fc67d8bd48e33349ffe976b6e4 at: https://64e64c3ef7d9ad04784e5bae--layer5.netlify.app

File: src/collections/events/IstioCon 2023/index.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: "IstioCon Virtual 2023"
+date: 2023-09-25
+eurl: "https://events.linuxfoundation.org/istiocon/"
+thumbnail: ./IstioCon2023.png
+darkthumbnail: ./IstioCon2023.png
+type: Event
+published: true
```
Comment (Chadha93) [inline src/collections/events/IstioCon 2023/index.mdx:8]:
```suggestion
published: true
resource: true
```

**Comment by l5io** [comment]
🚀 Preview for commit b9ae0de2479dad8e01b87d8cb01b193d32569d1f at: https://64e657b0551e6411acc0d812--layer5.netlify.app

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**